src/instructlab/data/generate.py

```diff
@@ -210,6 +211,27 @@ def generate(
         except Exception as exc:
             click.secho(f"Failed to start server: {exc}", fg="red")
             raise click.exceptions.Exit(1)
+
+    # figure out what model is running
```

Comment:
I believe the logic should be identical to what chat was doing before this patch.